services/clamav-rest-client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@planning-inspectorate/pins-clamav",
```

Comment:
pins-clamav-rest-client?

Reply:
Bit worried about changing this because forms web app and lpa questionnaire depend on this name. And I don't fully understand what the impact would be of changing this in the npmjs package manager.

Reply:
Glad we sorted this now and created a new package in the npm repo